packages/animations/lib/src/fade_scale_transition.dart

```diff
@@ -123,6 +123,7 @@ class FadeScaleTransition extends StatelessWidget {
   const FadeScaleTransition({
     Key key,
     @required this.animation,
+    this.alignment = Alignment.center,
```

Comment:
Update documentation above that alignment cannot be null and add an assert below.